sa/saro.go

```diff
@@ -751,6 +779,12 @@ func (ssa *SQLStorageAuthorityRO) GetAuthorization2(ctx context.Context, req *sa
 		// read replica yet. If we get a NoRows, wait a little bit and retry, once.
 		ssa.clk.Sleep(ssa.lagFactor)
 		obj, err = ssa.dbReadOnlyMap.Get(authzModel{}, req.Id)
+		if obj == nil || err != nil {
```

Comment:
I think this can just be:

```suggestion
		if err != nil {
```